ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/ChainGenerator/Tests/BitVector.hs

```diff
@@ -141,3 +158,56 @@ prop_findIthZeroInV testSetup = case testSetup of
                       $ QC.counterexample "There are not testIndex-many post-polarization-0s preceding i!"
                       $ targetsInPrecedingWords QC.=== C.getCount testIndex
                   )
+
+
+data FillInWindowSetup =
+    FillInWindowSetup
+      SomePol
+      Int -- ^ k
+      Int -- ^ size of the window to fill
+      QCGen -- ^ random generator
+      (V.Vector S) -- ^ the vector to fill
+  deriving (Show)
+
+instance QC.Arbitrary FillInWindowSetup where
+  arbitrary = do
+      pol <- QC.arbitrary
+      QC.NonNegative k <- QC.arbitrary
+      QC.NonNegative k1 <- QC.arbitrary
+      QC.NonNegative k2 <- QC.arbitrary
+      qcGen <- QC.arbitrary
+      let szV = k1 + 1
```

Comment:
Any particular reason for not testing `szV == 0`? It seems to work fine.

Reply:
Changed to `szV = k1`.